wizard/src/utils/detect_place.ts

```diff
@@ -74,13 +82,42 @@ export class PlaceDetector {
     }
   }
 
+  // The low confidence column detector simply checks if the column header
+  // (string) matches one of the supported place strings in this.placeTypes.
+  // The header is converted to lower case and only alphanumeric chars are used.
+  // If there is no match, the return value is null.
+  detectLowConfidence(header: string): string {
+    var h = header.toLowerCase().replace(/[^a-z0-9]/gi,'')
+
+    if (this.placeTypes.has(h)) {
+      var place = h.charAt(0).toUpperCase() + h.substr(1);
+
+      // "LatLon" needs special care after the processing above.
+      if (place == "Latlon") {
+        return "LatLon";
+      }
+      return place;
+    }
+    return null;
+  }
+
   // Detecting Place.
```

Comment:
function comments should look like: https://github.com/datacommonsorg/website/blob/master/static/js/chart/draw_choropleth.ts#L707-L717

Reply:
Got it, thanks!